packages/components/app/styles/components/page-header.scss

```diff
@@ -44,6 +44,15 @@
     justify-content: space-between;
     min-width: 0; // this is important or it will blow beyond the parent flexbox width with long non-breaking names
   }
+
+  // Dropdowns with @listPosition="bottom-right" need an override to prevent them from spilling outside the page
```

Comment:
```suggestion
  // Dropdowns with @listPosition="bottom-right" need an override to prevent them from spilling outside the page
  // Likely we will be be able to remove this override once we introduce collision detection for the dropdown - see: https://hashicorp.atlassian.net/browse/HDS-1995
```